tests/inspect/inspect.go

```diff
@@ -78,6 +79,7 @@ var (
 		SilentSigterm      bool
 		CheckMountNS       bool
 		PrintNoNewPrivs    bool
+		CheckMknod         string
```

Comment:
This is mostly bikeshedding, but I would remove the "check" prefix for this feature, as it is not really checking anything, just doing a plain mknod.